src/client/tile.cpp

```diff
@@ -1124,6 +1122,15 @@ video::IImage* TextureSource::generateImage(const std::string &name)
  * @param driver driver to use for image operations
  * @return image or copy of image aligned to npot2
  */
+
+inline u16 get_GL_major_version()
+{
+	const GLubyte *gl_version = glGetString(GL_VERSION);
+	std::string gl_ver((const char *)gl_version);
+	u16 GL_version = stoi(gl_ver.substr(0,1));
+	return GL_version;
```

Comment:
I know very well that this makes no different perf-wise, but since it's just 1 char you could just:
`return (u16) (gl_version[0] - '0');`